README.md

```diff
@@ -347,6 +348,43 @@ certutil -hashfile <file> SHA1
 Known good *SHA1* checksum of the Darkflame Universe client:
 - `91498e09b83ce69f46baf9e521d48f23fe502985` (packed client, zip compressed)
 
+# Docker
+
+## Standalone
+
+For standalone deployment, you can use the image provided via Github's Container Repository:
+`ghcr.io/darkflameuniverse/darkfameserver`
+
+This assumes that you have a database deployed to your host or in another docker container.
+
+A basic deployment of this contianer would look like:
+```sh
+# example docker contianer deployment
+docker run -it \
+    -v /path/to/configs/:/app/configs \
+    -v /path/to/logs/:/app/logs \
+    -v /path/to/dumps/:/app/dumps \
+    -v /path/to/res:/app/res:ro \
+    -v /path/to/resServer:/app/resServer \
+    -e DLU_CONFIG_DIR=/app/configs \
+    -e DUMP_FOLDER=/app/dumps \
+    -p 1001:1001/udp \
+    -p 2005:2005/udp \
+    -p 3000-3300:3000-3300/udp \
+ghcr.io/darkflameuniverse/darkfameserver:latest
+```
+You will need to replace the `/path/to/`'s to reflect the paths on your host.
+
+Any config option in the `.ini`'s can be overridden with environmental variables: Ex: `log_to_console=1` from `shared_config.ini` would be overidden like  `-e LOG_TO_CONSOLE=0`
```

Comment:
Maybe mention that all servers will pick it up, so service specific settings may have trouble and should use `{world,chat,auth,master}config.ini`

Reply:
Ah neat, you fixed the cases where that would have been an issue (port)

Reply:
oh yeah, I think i put that in the PR description when opening it for review